### PR TITLE
Show Just 1 Alert Banner for ERC20 Token

### LINKED ISSUE #1566: Show Just 1 Alert Banner for ERC20 Token
Two alert banners show when i've looked at two ERC 20 tokens, and not activated it.

We should show just 1 alert banner.

screen shot 2018-06-27 at 7 14 17 pm

### PR DESCRIPTION
**Pull request by pinkiebell**
Fixes #1566

### CODE REVIEW

File: app/assets/v2/js/shared.js [hunk 1]
```diff
@@ -825,7 +825,11 @@ var promptForAuth = function(event) {
 
     token_contract.allowance.call(from, to, function(error, result) {
       if (error || result.toNumber() == 0) {
-        _alert("You have not yet enabled this token.  To enable this token, go to the <a style='padding-left:5px;' href='/settings/tokens'> Token Settings page and enable it</a>. (this is only needed one time per token)");
+        if (!document.alert_enable_token_shown) {
+          _alert("You have not yet enabled this token.  To enable this token, go to the <a style='padding-left:5px;' href='/settings/tokens'> Token Settings page and enable it</a>. (this is only needed one time per token)");
+        }
+        document.alert_enable_token_shown = true;
```
Comment (thelostone-mc) [inline app/assets/v2/js/shared.js]: I think it should be a warning alert like [here](https://github.com/gitcoinco/web/blob/639b0de163aa2a7d251efe6d605492264e9c9b6f/app/assets/onepager/js/send.js#L115)

**Comment by pinkiebell** [comment]
Ahh, will rebase ;)

**Comment by thelostone-mc** [comment]
@pinkiebell could you squash the commits into one neat commit  ? 
Also what's the content of `error.toString()` 
Need to make sure we aren't showing info that's not needed for the user

**Comment by pinkiebell** [comment]
@thelostone-mc 
Hopefully a sophisticated error message. But I really don't know, the error object is always null no matter what I pass. However that might change in the future and could provide more info is something unexpected happens.

Let me know if you want me to revert back to just the warning message.

**Comment by pinkiebell** [comment]
@thelostone-mc 
https://github.com/ethereum/web3.js/blob/54e40bb4bbb97ba9590c3ba6f933951c7a04dd16/lib/web3/errors.js

**Comment by thelostone-mc** [comment]
Let's just stick to warning for now and do away with displaying the error message 👍

**Comment by pinkiebell** [comment]
@thelostone-mc 
I pushed a little update, is this good to go?

**Pull request merged**